UnitTests/BraintreeVisaCheckoutTests/BTConfiguration+VisaCheckout_Tests.swift

```diff
@@ -0,0 +1,112 @@
+import XCTest
+import VisaCheckoutSDK
+import BraintreeTestShared
+@testable import BraintreeCore
+@testable import BraintreeVisaCheckout
+
+class BTConfiguration_VisaCheckoutTests: XCTestCase {
+    
+    func testIsVisaCheckoutEnabled_whenVisaCheckoutEmptyJSONExists_returnsFalse() {
+        let configurationJSON = BTJSON(value: [
+            "visaCheckout": [:]
+        ])
+        let configuration = BTConfiguration(json: configurationJSON)
+        
+        XCTAssertFalse(configuration.isVisaCheckoutEnabled)
+    }
+    
+    func testIsVisaCheckoutEnabled_whenVisaCheckoutFromConfigurationJSONExists_returnsTrue() {
+        let configurationJSON = BTJSON(value: [
+            "visaCheckout": [
+                "apiKey": "apikey" // key must be 'apiKey' not 'apikey' to match Swift logic
+            ]
+        ])
+        let configuration = BTConfiguration(json: configurationJSON)
+        
+        XCTAssertTrue(configuration.isVisaCheckoutEnabled)
+    }
+    
+    func testIsVisaCheckoutEnabled_whenVisaCheckoutFromConfigurationJSONIsMissing() {
+        let configurationJSON = BTJSON(value: [:])
+        let configuration = BTConfiguration(json: configurationJSON)
+        
+        XCTAssertFalse(configuration.isVisaCheckoutEnabled)
+    }
+    
+    func testVisaCheckoutSupportedNetworks_returnsSupportedNetworks() {
+        let configurationJSON = BTJSON(value: [
+            "visaCheckout": [
+                "supportedCardTypes": [
+                    "Visa",
+                    "MasterCard",
+                    "American Express",
+                    "Discover"
+                ]
+            ]
+        ])
+        let configuration = BTConfiguration(json: configurationJSON)
+        
+        XCTAssertEqual(configuration.acceptedCardBrands, [CardBrand.visa.rawValue, CardBrand.mastercard.rawValue, CardBrand.amex.rawValue, CardBrand.discover.rawValue])
+    }
+    
+    func testVisaCheckoutSupportedNetworks_whenSupportedCardTypesContainsUnknownNetworks_doesNotIncludeThem() {
+        let configurationJSON = BTJSON(value: [
+            "visaCheckout": [
+                "supportedCardTypes": [
+                    "Visa",
+                    "MasterCard",
+                    "American Express",
+                    "Discover",
+                    "MysteryCard",
+                    "Braintree Express"
+                ]
+            ]
+        ])
+        let configuration = BTConfiguration(json: configurationJSON)
+        
+        XCTAssertEqual(configuration.acceptedCardBrands, [CardBrand.visa.rawValue, CardBrand.mastercard.rawValue, CardBrand.amex.rawValue, CardBrand.discover.rawValue])
```

Comment:
Should we be checking this against the literal strings instead? Something like this with the actual values:
```suggestion
        XCTAssertEqual(configuration.acceptedCardBrands, ["visa" , "mastercard", "amex", "discover"])
```

Reply:
hmm i think it's okay since it looks like `acceptedCardBrands` is of type`[Int]` and `supportedCardTypesToAcceptedCardBrands` takes in the string array of these card types and converts them into the accepted card brands [here](https://github.com/braintree/braintree_ios/blob/76adaf3db0873df2dda53f86837ef1fa976748e4/Sources/BraintreeVisaCheckout/BTConfiguration%2BVisaCheckout.swift#L42)